ilc/common/router/Router.js

```diff
@@ -55,7 +55,7 @@ module.exports = class Router {
         });
     }
 
-    __compiler(routes) {
+    #compiler = (routes) => {
```

Comment:
I think there was some issue with private class properties & babel in context of an old browsers. 
@oleh-momot do you remember it?

Reply:
Looks like private props will be complied onto a thing using WeakSet... Which is not supported in IE.
https://babeljs.io/docs/en/babel-plugin-proposal-private-methods
https://caniuse.com/#search=Weakset